package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "@moxy/next-with-router-ref",
+  "version": "1.0.0",
```

Comment:
Isn't better to start with `0.0.0` and then release it as major?

Reply:
fixed!